science/PothosLiquidDSP/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           cxx11 1.1
+
+platforms           darwin macosx
+categories          science
+license             Boost-1
+maintainers         {@ra1nb0w irh.it:rainbow} {michaelld @michaelld} openmaintainer
+
+description         Pothos toolkit for liquid DSP processing cores
+long_description    This toolkit wraps Pothos blocks around the Liquid \
+    DSP library. Blocks are generated from the liquid DSP include header \
+    and a file describing the available functions and parameters. \
+
+github.setup        pothosware PothosLiquidDSP 98b157a0d064fed19df05179172376ea15b0d31b
+version             0.0.0-git-20190409
+checksums           rmd160  fc9b1e4bc33594f917f5bb16babf4e42c7a36e9e \
+                    sha256  b0aab1b61b838f4e7b764b0e32b76a2e9f961179c1ff58454043a80b6dd3bbc2 \
+                    size    37459
+revision            0
+
+set python_version  36
+
+depends_build-append \
+    port:py${python_version}-ply \
+    port:py${python_version}-yaml \
+    port:py${python_version}-mako
+
+depends_lib-append \
+    port:PothosCore \
+    port:liquid-dsp
```

Comment:
I can't get this port to build with the latest `liquid-dsp` port (committing an update shortly). The warnings are errors are all like these 2 ... we can, of course, ignore the errors, but I'd be curious if they can be dealt with ... not sure if the issue here is with this port of `liquid-dsp` ...
```
:info:build [ 82%] Building CXX object CMakeFiles/LiquidBlocks.dir/iirdecim.cpp.o
:info:build /usr/bin/clang++  -DLiquidBlocks_EXPORTS  -pipe -Os -DNDEBUG -I/opt/local/include -stdlib=libc++ -isysroot/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk -mmacosx-version-min=10.14 -fPIC   -std=gnu++11 -o CMakeFiles/LiquidBlocks.dir/iirdecim.cpp.o -c /opt/local/var/macports/build/_opt_sources_MacPorts_ports_github_macports_science_PothosLiquidDSP/PothosLiquidDSP/work/build/iirdecim.cpp
:info:build In file included from /opt/local/var/macports/build/_opt_sources_MacPorts_ports_github_macports_science_PothosLiquidDSP/PothosLiquidDSP/work/build/freqmodem.cpp:9:
:info:build /opt/local/include/liquid/liquid.h:6030:24: warning: 'polyc_val' has C-linkage specified, but returns user-defined type 'liquid_double_complex' (aka 'complex<double>') which is incompatible with C [-Wreturn-type-c-linkage]
:info:build LIQUID_POLY_DEFINE_API(LIQUID_POLY_MANGLE_CDOUBLE,
:info:build                        ^
:info:build /opt/local/var/macports/build/_opt_sources_MacPorts_ports_github_macports_science_PothosLiquidDSP/PothosLiquidDSP/work/build/firhilbf.cpp:176:13: error: no matching function for call to 'firhilbf_c2r_execute'
:info:build             firhilbf_c2r_execute(_q, *inxBuff, outyBuff);
:info:build             ^~~~~~~~~~~~~~~~~~~~
:info:build /opt/local/include/liquid/liquid.h:2702:27: note: candidate function not viable: requires 4 arguments, but 3 were provided
:info:build LIQUID_FIRHILB_DEFINE_API(LIQUID_FIRHILB_MANGLE_FLOAT, float, liquid_float_complex)
:info:build                           ^
```

Reply:
fixed upstream. Now build fine.